### ISSUE TITLE
visual comparison of two different parameter files to easily see which parameters changed

### ISSUE THREAD

**Issue by stephanie-r-jones** [jonescompneurolab/hnn#221]
In developing and testing hypotheses on the circuit origin of the data, it would be very useful for the user to have an easy way to see which parameters are different across different simulations, perhaps as sliders similar to the optimization dialog window.

**Comment by jasmainak** [comment]
what about parameters which are added or deleted? Do you want them to be part of the visualization?

**Comment by stephanie-r-jones** [comment]
Do you mean parameters that are added or deleted as the code expands? The parameters of interest would be those describing the local network connectivity, and the exogenous drive. For ERP simulations,  then optimization tool optimizes over several parameters and these in particular would be important to visualize before and after optimization to see what changed.

**Comment by jasmainak** [comment]
no actually I was asking a more basic question. Since two parameter files don't always have the same set of parameters, I was asking whether the visualization would show only the common paramters or a set of predecided parameters that were also available in the files

**Comment by rythorpe** [comment]
This might help clarify the issue that @jasmainak is bringing up. Plotted below are the evoked input parameters pertaining to two different simulations. Note that the param file corresponding to the bottom image doesn't include values associated with the distal inputs that are in the top image.

![laser_2020_02_12_3prox_4dist_params](https://user-images.githubusercontent.com/20212206/91349397-3b672880-e7b3-11ea-9550-35275f57863a.jpeg)

![laser_2020_04_08_3prox_params](https://user-images.githubusercontent.com/20212206/91349340-2a1e1c00-e7b3-11ea-9d42-102d9d8ed75c.jpeg)

**Comment by rythorpe** [comment]
The function that generated the plots above received an argument where the user specified which evoked inputs it would look for in both param files. This way, both figures would be side-by-side comparable. This should be relatively simple to implement in `hnn_core.viz` once we have refactored the `hnn_core.feed` module.

**Comment by stephanie-r-jones** [comment]
Ahh... I understand. I like these plots. However, rather than have the user have to input anything more specific other than the file names to compare, could the function just plot ALL parameters in EITHER file, and if a parameter is only in one file mark it as NAN in the corresponding space in the other file's plot?

**Comment by jasmainak** [comment]
One issue with plotting all the parameters is that the units and scale of different parameters may be vastly different. But we need to try and see how that looks ...

**Comment by rythorpe** [comment]
> One issue with plotting all the parameters is that the units and scale of different parameters may be vastly different.

This is true. In my situation I only plotted the parameters related to the evoked inputs, however, there are many other types of parameters and we will need to be creative with how we present them.

**Comment by jasmainak** [comment]
also the input times presented this way are not very meaningful in my opinion, it's better presented on the plot of the dipole. One idea could be to show the parameters such as `gbar` as connection strengths on a 3D "connectivity plot".

**Comment by jasmainak** [comment]
or a circle graph: https://mne.tools/stable/_images/sphx_glr_plot_mixed_source_space_connectivity_001.png
